modules/op-adding-triggers.adoc

```diff
@@ -81,16 +83,48 @@ spec:
           claimName: source-pvc
 ----
 
-. Create the `TriggerTemplate`:
+. Create the `TriggerTemplate` resource:
 +
+[source,terminal]
 ----
 $ oc create -f <triggertemplate-yaml-file-name.yaml>
 ----
 +
-Alternatively, you can create the `TriggerTemplate` directly from the `pipelines-tutorial` Git repository:
+Alternatively, you can create the `TriggerTemplate` resource directly from the `pipelines-tutorial` Git repository:
++
+[source,terminal,subs="attributes+"]
+----
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/03_triggers/02_template.yaml
+----
+
+. Copy the contents of the following sample `Trigger` YAML file and save it:
++
+[source,yaml]
+----
+apiVersion: triggers.tekton.dev/v1alpha1 <1>
```

Comment:
Now I realized that we have these number references in this code snippet but they are not explained anywhere.

Reply:
Good catch, Pavol! Will update asap!

Reply:
@ppitonak the explanations are there in the the About Triggers module. For this module i have removed the callouts and made it consistent with the other snippets. PTAL, Thank you.